### PR TITLE
2.11 Screenshots

### PR DESCRIPTION
**Pull request by browkin**
* ios de & en - 2. Screenshot von 2.10 auf 2.11 übertragen da Blau statt Grau.

### CODE REVIEW

File: src/data/screenshots.json [hunk 1]
```diff
@@ -219,201 +219,201 @@
...
             {
                 "title": "Certificates",
                 "anchor": "ios_certificates",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_health_certificate_empty_screen", "alt": "Certificates image 1 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_health_certificate_consent_screen", "alt": "Certificates image 2 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_family_certificate-cert-1", "alt": "Certificates image 3 of 4" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_family_certificate-cert-2", "alt": "Certificates image 4 of 4" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_health_certificate_empty_screen", "alt": "Certificates image 1 of 4" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_health_certificate_consent_screen", "alt": "Certificates image 2 of 4" },
```
Comment (Ein-Tim) [inline src/data/screenshots.json:355]: Image one and two should be flipped, the app first shows the consent screen and after clicking on "Continue" the app shows the "Empty certificates tab" screen.

File: src/data/screenshots.json [hunk 2]
```diff
@@ -219,201 +219,201 @@
...
                 ]
             },
             {
                 "title": "Vaccination Certificate",
                 "anchor": "ios_vaccination_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Vaccination certificate image 1 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part1", "alt": "Vaccination certificate image 2 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part2", "alt": "Vaccination certificate image 3 of 3" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Vaccination certificate image 1 of 3" },
```
Comment (Ein-Tim) [inline src/data/screenshots.json:375]: This screenshot was not taken on version 2.11, it should be replaced with a screenshot taken on version 2.11.

File: src/data/screenshots.json [hunk 3]
```diff
@@ -219,201 +219,201 @@
...
                 ]
             },
             {
                 "title": "Invalid Certificates",
                 "anchor": "ios_invalid_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Invalid certificate image 1 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part1", "alt": "Invalid certificate image 2 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part2", "alt": "Invalid certificate image 3 of 3" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Invalid certificate image 1 of 3" },
```
Comment (Ein-Tim) [inline src/data/screenshots.json:414]: This screenshot has not been taken on version 2.11. It should be removed, as the screenshot which follows is just a newer version of it. 

```suggestion
```

**Don't forget to change the counting "Invalid certificate image x of 2"**

File: src/data/screenshots_de.json [hunk 4]
```diff
@@ -24,190 +25,190 @@
...
               },
               {
                   "title": "Zertifikate vor einer Reise auf Gültigkeit prüfen",
                   "anchor": "android_check_validity_certificate",
                   "images": [
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Zertifikate auf Gültigkeit prüfen Bild 1 von 4" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_validation_date_selection", "alt": "Zertifikate auf Gültigkeit prüfen Bild 2 von 4" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_validation_country_selection", "alt": "Zertifikate auf Gültigkeit prüfen Bild 3 von 4" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_validation", "alt": "Zertifikate auf Gültigkeit prüfen Bild 4 von 4" }
+                      { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Zertifikate auf Gültigkeit prüfen Bild 1 von 4" },
```
Comment (Ein-Tim) [inline src/data/screenshots_de.json:199]: This iOS screenshot has been taken on version 2.10 and not on version 2.11. The info text reg. CovPassCheck is missing.

It should be replaced with a screenshot taken on version 2.11.

**Comment by Ein-Tim** [comment]
@dsarkar Would a PR reg. my suggested changes be welcome? Or would you prefer to do the changes yourself (e.g. the Open Source Team)?

**Comment by dsarkar** [comment]
@Ein-Tim Thanks for the review. Do you want to crate a complete new PR ro can we commit you suggestion (where applicable?)

File: src/data/screenshots_de.json [hunk 5]
```diff
@@ -5,14 +5,15 @@
...
-                "Neue Funktionen in Version 2.10 sind Mitteilungen zu Auffrischungsimpfungen und das Drucken von Impfzertifikaten.",
-                "Die App kann Ihnen nun eine Mitteilung anzeigen, die Sie über eine eventuell anstehende Auffrischungsimpfung informiert.",
-                "Sie können nun eine Druckversion Ihrer Impfzertifikate anzeigen, die es Ihnen ermöglicht, die Zertifikate als PDF-Dokument zu speichern oder auszudrucken.",
+                "Verschaffen Sie sich mithilfe von Screenshots einen Überblick über die Funktionen, die die Corona-Warn-App mit Version 2.11 (verfügbar seit 06.10.2021) bietet. Bitte beachten Sie, dass die Screenshots für Android und iOS designbedingt leicht voneinander abweichen können.",
+                "Neue Funktionen in Version 2.11 sind ein universeller QR-Code-Scanner, die Möglichkeit, QR-Codes zu importieren und Anpassungen der Kontaktdauer.",
+                "Universeller QR-Code Scanner: Sie können nun unabhängig davon, von wo ein QR-Code-Scanner in der Corona-Warn-App geöffnet wird, alle von der App unterstützen Dokumente scannen: PCR-Tests, Schnelltests, Event Check-ins und Zertifikate",
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/screenshots_de.json]: @Ein-Tim Thanks.

File: src/data/screenshots.json [hunk 6]
```diff
@@ -5,14 +5,14 @@
...
-                "New features in version 2.10 are notifications about booster vaccinations and printing of vaccination certificates.",
-                "The app can now give you a notification informing you of any upcoming booster vaccination.",
-                "You can now display a printable version of your vaccination certificates, which allows you to save or print the certificates as a PDF document.",
+                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.11 (available since 06.10.2021). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
+                "New features in version 2.11 includes an universal QR code scanner, the ability to import QR codes and adjustments to the contact duration.",
+                "Universal QR code scanner: you can now scan all documents supported by the app regardless of where a QR code scanner is opened from in the Corona-Warn-App: PCR tests, rapid tests, event check-ins and certificates.",
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/screenshots.json:14]: @Ein-Tim Thanks.

File: src/data/screenshots.json [hunk 7]
```diff
@@ -5,14 +5,14 @@
...
         },
         "content": {
             "textblock": [
-                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.10 (available since 22.09.2021). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
-                "New features in version 2.10 are notifications about booster vaccinations and printing of vaccination certificates.",
-                "The app can now give you a notification informing you of any upcoming booster vaccination.",
-                "You can now display a printable version of your vaccination certificates, which allows you to save or print the certificates as a PDF document.",
+                "Browse through the screenshots to get an overview of the features offered by the Corona-Warn-App with the release of Version 2.11 (available since 06.10.2021). Please note that the screenshots may vary slightly between Android and iOS due to technical reasons.",
+                "New features in version 2.11 includes an universal QR code scanner, the ability to import QR codes and adjustments to the contact duration.",
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/screenshots.json:13]: @Ein-Tim ok, thanks

File: src/data/screenshots.json [hunk 8]
```diff
@@ -5,14 +5,14 @@
...
+                "Universal QR code scanner: you can now scan all documents supported by the app regardless of where a QR code scanner is opened from in the Corona-Warn-App: PCR tests, rapid tests, event check-ins and certificates.",
+                "Import QR code: on Android devices, you can now import QR codes from photos or PDF files via the QR Code Scanner",
```
[the remark this answers is not in the document]
Comment (dsarkar) [reply src/data/screenshots.json:15]: @Ein-Tim thanks

**Comment by Ein-Tim** [comment]
@dsarkar I'm quite busy now so probably best would be if you could commit the changes which can be commited easily and I will create a PR this weekend (after this PR has been merged).

Does this sound good to you?

File: src/data/screenshots_de.json [hunk 9]
```diff
@@ -219,201 +220,201 @@
...
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_test_certificate_overview_part1", "alt": "Testzertifikat Bild 3 von 5" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_test_certificate_valid_details_part1", "alt": "Testzertifikat Bild 4 von 5" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_test_certificate_valid_details_part2", "alt": "Testzertifikat Bild 5 von 5" }
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-homescreenrisk_level_low_installation_14days_0001", "alt": "Testzertifikat Bild 1 von 5" },
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-submissionflow_screenshot_test_certificate_entered_birthday-DE", "alt": "Testzertifikat Bild 2 von 5" },
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_test_certificate_overview_part1", "alt": "Testzertifikat Bild 3 von 5" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:367]: @Ein-Tim Thanks!

File: src/data/screenshots_de.json [hunk 10]
```diff
@@ -219,201 +220,201 @@
...
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_print_pdf_infoscreen", "alt": "Impfzertifikat drucken Bild 3 von 5" },
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_print_pdf_pdfscreen", "alt": "Impfzertifikat drucken Bild 4 von 5" },
+                    { "filename": "/assets/screenshots/2-11/de/ios/print_share", "alt": "Impfzertifikat drucken Bild 5 von 5" }
                 ]
             },
             {
                 "title": "Genesenenzertifikat",
                 "anchor": "ios_recovery_certificate",
                 "images": [
                     { "filename": "/assets/screenshots/2-6/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Genesenenzertifikat Bild 1 von 3" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:396]: @Ein-Tim Thanks!

**Comment by dsarkar** [comment]
@Ein-Tim We check your suggestions and will commit. This will be merged then into branch sarkar-release-2-11. (And probably into master on Wednesday morning) Thanks.

File: src/data/screenshots_de.json [hunk 11]
```diff
@@ -219,201 +220,201 @@
...
                 ]
             },
             {
                 "title": "Ungültige Zertifikate",
                 "anchor": "ios_invalid_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Ungültige Zertifikate Bild 1 von 3" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part1", "alt": "Ungültige Zertifikate Bild 2 von 3" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part2", "alt": "Ungültige Zertifikate Bild 3 von 3" }
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Ungültige Zertifikate Bild 1 von 3" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:413]: @Ein-Tim Thanks!

File: src/data/screenshots.json (hunk 3 above)
Comment (browkin) [reply src/data/screenshots.json:414]: @Ein-Tim Thanks!

File: src/data/screenshots_de.json [hunk 12]
```diff
@@ -24,190 +25,190 @@
...
                   ]
               },
               {
                   "title": "Ungültige Zertifikate",
                   "anchor": "android_invalid_certificate",
                   "images": [
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Ungültige Zertifikate Bild 1 von 3" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part1", "alt": "Ungültige Zertifikate Bild 2 von 3" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part2", "alt": "Ungültige Zertifikate Bild 3 von 3" }
+                      { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Ungültige Zertifikate Bild 1 von 3" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:209]: @Ein-Tim Thanks!

File: src/data/screenshots_de.json [hunk 13]
```diff
@@ -24,190 +25,190 @@
...
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/de/android/PersonOverviewFragment_empty", "alt": "Zertifikate Bild 1 von 6"},
-                    { "filename": "/assets/screenshots/2-10/de/android/CovidCertificateOnboardingFragment", "alt": "Zertifikate Bild 2 von 6"},
-                    { "filename": "/assets/screenshots/2-10/de/android/PersonOverviewFragment_one_person", "alt": "Zertifikate Bild 3 von 6"},
-                    { "filename": "/assets/screenshots/2-10/de/android/PersonOverviewFragment_many_persons", "alt": "Zertifikate Bild 4 von 6"},
-                    { "filename": "/assets/screenshots/2-10/de/android/PersonDetailsFragment_cwa", "alt": "Zertifikate Bild 5 von 6"},
-                    { "filename": "/assets/screenshots/2-10/de/android/PersonDetailsFragment_cwa_2", "alt": "Zertifikate Bild 6 von 6"}
+                    { "filename": "/assets/screenshots/2-11/de/android/PersonOverviewFragment_empty", "alt": "Zertifikate Bild 1 von 6"},
+                    { "filename": "/assets/screenshots/2-11/de/android/CovidCertificateOnboardingFragment", "alt": "Zertifikate Bild 2 von 6"},
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:150]: @Ein-Tim Thanks!

File: src/data/screenshots.json [hunk 14]
```diff
@@ -24,190 +24,190 @@
...
                 "title": "Warn for Others",
                 "anchor": "android_warn_for_others",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/android/TestCertificateDetailsFragment", "alt": "Warn for others image 1 of 6" },
-                    { "filename": "/assets/screenshots/2-10/en/android/TraceLocationWarnInfoFragment", "alt": "Warn for others image 2 of 6" },
-                    { "filename": "/assets/screenshots/2-10/en/android/TraceLocationSelectionFragment", "alt": "Warn for others image 3 of 6" },
-                    { "filename": "/assets/screenshots/2-10/en/android/TraceLocationWarnDurationFragment", "alt": "Warn for others image 4 of 6" },
-                    { "filename": "/assets/screenshots/2-10/en/android/TraceLocationWarnTanFragment", "alt": "Warn for others image 5 of 6" },
-                    { "filename": "/assets/screenshots/2-10/en/android/TraceLocationOrganizerThanksFragment", "alt": "Warn for others image 6 of 6" }
+                    { "filename": "/assets/screenshots/2-11/en/android/TestCertificateDetailsFragment", "alt": "Warn for others image 1 of 6" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots.json:109]: @Ein-Tim Thanks!

File: src/data/screenshots_de.json [hunk 15]
```diff
@@ -219,201 +220,201 @@
...
                 ]
             },
             {
                 "title": "Impfzertifikat",
                 "anchor": "ios_vaccination_certificate",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Impfzertifikat Bild 1 von 3" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part1", "alt": "Impfzertifikat Bild 2 von 3" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part2", "alt": "Impfzertifikat Bild 3 von 3" }
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Impfzertifikat Bild 1 von 3" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:373]: @Ein-Tim Thanks you, added!

File: src/data/screenshots_de.json [hunk 16]
```diff
@@ -219,201 +220,201 @@
...
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part1", "alt": "Impfzertifikat Bild 2 von 3" },
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part2", "alt": "Impfzertifikat Bild 3 von 3" }
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:375]: @Ein-Tim Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots_de.json [hunk 17]
```diff
@@ -24,190 +25,190 @@
...
                   "anchor": "android_invalid_certificate",
                   "images": [
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Ungültige Zertifikate Bild 1 von 3" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part1", "alt": "Ungültige Zertifikate Bild 2 von 3" },
-                      { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part2", "alt": "Ungültige Zertifikate Bild 3 von 3" }
+                      { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_overview_part1", "alt": "Ungültige Zertifikate Bild 1 von 3" },
+                      { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part1", "alt": "Ungültige Zertifikate Bild 2 von 3" },
+                      { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_recovery_certificate_expired_details_part2", "alt": "Ungültige Zertifikate Bild 3 von 3" }
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json]: @Ein-Tim Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots.json [hunk 18]
```diff
@@ -24,190 +24,190 @@
...
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_my_qr_codes", "alt": "Check In image 1 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_create_qr_code_category_selection", "alt": "Check In image 2 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_create_qr_code_attributes", "alt": "Check In image 3 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_display_qr_code", "alt": "Check In image 4 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_qr_code_print_version", "alt": "Check In image 5 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_attendee_view_my_check_ins", "alt": "Check In image 6 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In image 7 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_attendee_view_contact_journal", "alt": "Check In image 8 of 8" }
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots.json:70]: @Ein-Tim  Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots.json [hunk 19]
```diff
@@ -219,201 +219,201 @@
...
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Vaccination certificate image 1 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part1", "alt": "Vaccination certificate image 2 of 3" },
-                    { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part2", "alt": "Vaccination certificate image 3 of 3" }
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_overview_part1", "alt": "Vaccination certificate image 1 of 3" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part1", "alt": "Vaccination certificate image 2 of 3" },
+                    { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_second_vaccination_certificate_details_part2", "alt": "Vaccination certificate image 3 of 3" }
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots.json:374]: @Ein-Tim Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots_de.json [hunk 20]
```diff
@@ -24,190 +25,190 @@
...
-                    { "filename": "/assets/screenshots/2-10/de/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In Bild 7 von 8" },
-                    { "filename": "/assets/screenshots/2-10/de/android/check_in_attendee_view_contact_journal", "alt": "Check In Bild 8 von 8" }
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_host_view_my_qr_codes", "alt": "Check In Bild 1 von 8" },
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_host_view_create_qr_code_category_selection", "alt": "Check In Bild 2 von 8" },
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_host_view_create_qr_code_attributes", "alt": "Check In Bild 3 von 8" },
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_host_view_display_qr_code", "alt": "Check In Bild 4 von 8" },
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_host_view_qr_code_print_version", "alt": "Check In Bild 5 von 8" },
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_attendee_view_my_check_ins", "alt": "Check In Bild 6 von 8" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:69]: @Ein-Tim Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots_de.json [hunk 21]
```diff
@@ -24,190 +25,190 @@
...
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In Bild 7 von 8" },
+                    { "filename": "/assets/screenshots/2-11/de/android/check_in_attendee_view_contact_journal", "alt": "Check In Bild 8 von 8" }
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:71]: @Ein-Tim Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots_de.json [hunk 22]
```diff
@@ -219,201 +220,201 @@
...
             {
                 "title": "Zertifikate",
                 "anchor": "ios_certificates",
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_health_certificate_empty_screen", "alt": "Zertifikate Bild 1 von 4" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_health_certificate_consent_screen", "alt": "Zertifikate Bild 2 von 4" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_family_certificate-cert-1", "alt": "Zertifikate Bild 3 von 4" },
-                    { "filename": "/assets/screenshots/2-10/de/ios/iPhone 11 Pro-screenshot_certificate_family_certificate-cert-2", "alt": "Zertifikate Bild 4 von 4" }
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_health_certificate_empty_screen", "alt": "Zertifikate Bild 1 von 4" },
+                    { "filename": "/assets/screenshots/2-11/de/ios/iPhone 11 Pro-screenshot_health_certificate_consent_screen", "alt": "Zertifikate Bild 2 von 4" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots_de.json:354]: @Ein-Tim Thanks!

File: src/data/screenshots.json [hunk 23]
```diff
@@ -24,190 +24,190 @@
...
-                    { "filename": "/assets/screenshots/2-10/en/android/check_in_attendee_view_check_in_detail_view", "alt": "Check In image 7 of 8" },
-                    { "filename": "/assets/screenshots/2-10/en/android/check_in_attendee_view_contact_journal", "alt": "Check In image 8 of 8" }
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_my_qr_codes", "alt": "Check In image 1 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_create_qr_code_category_selection", "alt": "Check In image 2 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_create_qr_code_attributes", "alt": "Check In image 3 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_display_qr_code", "alt": "Check In image 4 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_host_view_qr_code_print_version", "alt": "Check In image 5 of 8" },
+                    { "filename": "/assets/screenshots/2-11/en/android/check_in_attendee_view_my_check_ins", "alt": "Check In image 6 of 8" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots.json:68]: @Ein-Tim Thanks, we do not have them currently and send a request to the developer.

File: src/data/screenshots.json [hunk 24]
```diff
@@ -24,190 +24,190 @@
...
                 "images": [
-                    { "filename": "/assets/screenshots/2-10/en/android/PersonOverviewFragment_empty", "alt": "Certificates image 1 of 6"},
-                    { "filename": "/assets/screenshots/2-10/en/android/CovidCertificateOnboardingFragment", "alt": "Certificates image 2 of 6"},
-                    { "filename": "/assets/screenshots/2-10/en/android/PersonOverviewFragment_one_person", "alt": "Certificates image 3 of 6"},
-                    { "filename": "/assets/screenshots/2-10/en/android/PersonOverviewFragment_many_persons", "alt": "Certificates image 4 of 6"},
-                    { "filename": "/assets/screenshots/2-10/en/android/PersonDetailsFragment_cwa", "alt": "Certificates image 5 of 6"},
-                    { "filename": "/assets/screenshots/2-10/en/android/PersonDetailsFragment_cwa_2", "alt": "Certificates image 6 of 6"}
+                    { "filename": "/assets/screenshots/2-11/en/android/PersonOverviewFragment_empty", "alt": "Certificates image 1 of 6"},
+                    { "filename": "/assets/screenshots/2-11/en/android/CovidCertificateOnboardingFragment", "alt": "Certificates image 2 of 6"},
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots.json:148]: @Ein-Tim Thanks!

File: src/data/screenshots.json [hunk 25]
```diff
@@ -24,190 +24,190 @@
...
               },
               {
                   "title": "Check Certificates for Validity Before a Trip",
                   "anchor": "android_check_validity_certificate",
                   "images": [
-                      { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Certificate check image 1 of 4" },
-                      { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_validation_date_selection", "alt": "Certificate check image 2 of 4" },
-                      { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_validation_country_selection", "alt": "Certificate check image 3 of 4" },
-                      { "filename": "/assets/screenshots/2-10/en/ios/iPhone 11 Pro-screenshot_certificate_validation", "alt": "Certificate check image 4 of 4" }
+                      { "filename": "/assets/screenshots/2-11/en/ios/iPhone 11 Pro-screenshot_vaccination_certificate_valid_details_part3", "alt": "Certificate check image 1 of 4" },
```
[the remark this answers is not in the document]
Comment (browkin) [reply src/data/screenshots.json:197]: @Ein-Tim Thanks, added!

File: src/data/screenshots.json (hunk 1 above)
Comment (browkin) [reply src/data/screenshots.json:353]: @Ein-Tim Thanks!

File: src/data/screenshots_de.json (hunk 4 above)
Comment (browkin) [reply src/data/screenshots_de.json:198]: @Ein-Tim Thanks added!

**Pull request closed without merging**